lib/templates/deploy.batch.go.template

```diff
@@ -22,8 +13,23 @@ IF /I "%IN_PLACE_DEPLOYMENT%" EQU "1" (
 
 ECHO Prepare workspace
 IF "%GOROOT%" == "" (
-    :: select go version TODO
-    SET GOROOT=D:\Program Files\go\1.4.2
+  IF DEFINED GO_WEB_CONFIG_TEMPLATE (
+    :: Sort folder in reverse alphabetically order (-N), and folder only (G)
```

Comment:
right.. miss some test case.. my first attempt was to use another powershell script to do the SemVer comparison and output to a file.

but seems like once we spin out another process, we will lost the stdin stream. won`t able to read file anymore

e.g 
````
set /p myvar=<%pathToFIle%
````

let me dig a bit deeper